modules/button/react/lib/types.ts

```diff
@@ -45,6 +45,7 @@ export enum IconButtonVariant {
   CircleFilled = 'circleFilled',
   Inverse = 'inverse',
   InverseFilled = 'inverseFilled',
+  ToolbarSquare = 'toolbarSquare',
```

Comment:
Unless we're planning on adding a `ToolbarCircle`, we should just call this `Toolbar`